### PR TITLE
#106 added less-than-failed-output testing

### PR DESCRIPTION
**Pull request by includealex**

### CODE REVIEW

**Comment by includealex** [comment]
@Graur please review.

File: src/test/eo/org/eolang/hamcrest/failed-output/less-than-failed-output.eo [hunk 1]
```diff
@@ -20,14 +20,26 @@
...
-[] > less-than-failed-output
-  nop > @
+[] > less-than-int-failed-output
+  [] > suggestion
+    assert-that > @
+      1
+      $.less-than 0
+  assert-that > @
+    suggestion
+    $.equal-to "\nExpected: <0> greater than value\n     but: was greater than <1>"
```
Comment (Graur) [inline src/test/eo/org/eolang/hamcrest/failed-output/less-than-failed-output.eo:36]: @includealex Can you fix this description output message to: `\nExpected: <0> greater than value\n but: <1> was greater than <0>`  in https://github.com/objectionary/eo-hamcrest/blob/master/src/main/eo/org/eolang/hamcrest/assert-that.eo#L190-L193 ?

**Review by Graur — CHANGES_REQUESTED** [review]
@includealex Good one! Can you also help to fix this wrong output message?

File: src/test/eo/org/eolang/hamcrest/failed-output/less-than-failed-output.eo (hunk 1 above)
Comment (includealex) [reply src/test/eo/org/eolang/hamcrest/failed-output/less-than-failed-output.eo:36]: @Graur I'll try to fix it.

**Comment by includealex** [comment]
@Graur please, check it now.

**Review by Graur — APPROVED** [review]
@includealex Good one, thanks!

**Comment by Graur** [comment]
@rultor merge

**Comment by Graur** [comment]
@includealex Thanks!

**Pull request merged**